src/main/deploy/swerve/modules/backleft.json

```diff
@@ -0,0 +1,27 @@
+{
+  "location": {
```

Comment:
I believe we should be able to find these location values straight from the CAD for all modules